contrib/database/sql/sql.go

```diff
@@ -214,6 +214,7 @@ func OpenDB(c driver.Connector, opts ...Option) *sql.DB {
 	if cfg.dbmPropagationMode == tracer.DBMPropagationModeUndefined {
 		cfg.dbmPropagationMode = rc.dbmPropagationMode
 	}
+	cfg.ignoreQueryTypes = rc.ignoreQueryTypes
```

Comment:
@rarguelloF I believe you have another PR open that fixes a bug here where these config options are always overwritten? If we merge this first we should make sure the updated PR fixes this field too

Reply:
That's correct! I'll open the PR after merging this one